### PR TITLE
More flexibility with setting the release version for `./script/dock pkg`

### PR DESCRIPTION
**Pull request by o-fedorov**
* Allow to set RELEASE_VERSION and RELEASE_SUBVERSION for `./script/dock pkg` as environment variables.
* Fix an issue when the build script failed when RELEASE_VERSION contained metadata (symbols `-` and `+`, to be specific, like in `3.2.6-12`).

Related issue: https://github.com/percona/orchestrator/issues/36

### CODE REVIEW

**Comment by o-fedorov** [comment]
@egegunes, @fabio-silva, @kamil-holubicki, @igroene, could I please ask you to take care of this PR?

File: build.sh [hunk 1]
```diff
@@ -277,7 +281,7 @@ main() {
   local build_only=$5
   local build_path
 
-  if [ -z "${RELEASE_VERSION}" ] ; then
+  if [ "${RELEASE_VERSION}" == "" ] ; then
```
[the remark this answers is not in the document]
Comment (o-fedorov) [reply build.sh]: Thank you, updated.

File: build.sh [hunk 2]
```diff
@@ -228,8 +228,12 @@ package_linux() {
   [ $do_deb -eq 1 ] && fpm -v "${RELEASE_VERSION}" --epoch 1  -f -s dir -n orchestrator-client -m shlomi-noach --description "MySQL replication topology management and HA: client script" --url "https://github.com/openark/orchestrator" --vendor "GitHub" --license "Apache 2.0" -C $build_path/orchestrator-client --prefix=/ --depends 'jq >= 1.5' -t deb --deb-no-default-config-files .
 
   if [ ! -z "$package_name_extra" ] ; then
```
[the remark this answers is not in the document]
Comment (o-fedorov) [reply build.sh]: That's how this line of code looked initially. Anyway, updated.

**Review by kamil-holubicki — APPROVED** [review]

**Comment by o-fedorov** [comment]
Thank you for the approval. If this PR is good enough, could I please ask you to merge it? I don’t have rights to do it myself.

**Pull request merged**